go.mod

```diff
@@ -1,50 +1,50 @@
 module github.com/tinkerbell/pbnj
 
-go 1.18
+go 1.20
```

Comment:
We recently opted to adhere to the [literal definition of the `go` directive](https://go.dev/doc/modules/gomod-ref#go) and only update this when a language feature demands it.

Do we _need_ 1.20 features?

Reply:
No, there was no need for any Go 1.20 specific features,

In this case I bumped the Go version because of the newer linter having issues with the older Go version,
and the newer linter had other useful checks not available in the older version.

Reply:
Ok thanks.